components/shoplist-card/index.js

```diff
@@ -18,31 +18,67 @@ export const ShopCard = (props) => {
         <img src={path.join('./assets', product.image)} alt={product.name} />
       </Link>
       <div className="shop-card__content">
-        <p> {product.name}</p>
-        <p> {product.price}</p>
-        <button onClick={() => handleAddProduct(product.name)}> + </button>
-        <button onClick={() => handleRemoveProduct(product.name)}> - </button>
-        <p> Quantity : {props.quantity}</p>
+        <p className="shop-card-product-name"> {product.name}</p>
+        <p className="shop-card-product-price"> {product.price}</p>
+        <div className="shop-card-product-qnty">
+          <button onClick={() => handleAddProduct(product.name)}> + </button>
+          <span>{props.quantity}</span>
+          <button onClick={() => handleRemoveProduct(product.name)}> - </button>
+        </div>
```

Comment:
Can you please destructure the `props` and `product`.

Reply:
should I further destructure the product as it is already destructured from props?

Reply:
(props) can be named as ({product}) , and inside you can destructure for product that looks good in my view

Reply:
You can do something like this,
```
export const ShopCard = ({ product }) => {
  const { name, restOfTheFields } = product;
.
.
.
}
```

Reply:
we already had this on line number 5
```javascript
  const { product } = props;
```
That is why I was asking

I will convert it to 
```javascript
  const { product, quantity } = props;
```

Reply:
You also need to destructure the `product`

Reply:
Good work @agarwalamn

Reply:
<img src="https://i.imgur.com/jOqVRQ7.jpg"/>